py/tools/py/src/pth.rs

```diff
@@ -144,11 +148,45 @@ fn create_symlink(e: &DirEntry, root_dir: &Path, dst_dir: &Path) -> miette::Resu
             .and_then(|s| s.split_once(".runfiles/"))
             .map(|s| s.1)
             .unwrap();
-        return Err(
-            miette!(format!("site-packages/{}", path_to_conflict))
-            .wrap_err(format!("{}", next_conflict))
-            .wrap_err("Conflicting symlinks found when attempting to create venv. More than one package provides the file at these paths")
-        );
+
+        let d = MietteDiagnostic::new("Conflicting symlinks found when attempting to create venv. More than one package provides the file at these paths".to_string()).with_severity(severity);
+        let d2 = MietteDiagnostic::new(format!("site-packages/{}", path_to_conflict))
+            .with_severity(severity);
+        let d3 = MietteDiagnostic::new(next_conflict).with_severity(severity);
+
+        miette!(d3).wrap_err(d2).wrap_err(d)
+    }
+
+    if link.exists() {
+        // If the link already exists and is the same file, then there is no need to link this new one.
+        // Assume that if the files are the same, then there is no need to warn or error.
```

Comment:
Should this still warning even if they are the same file?

Reply:
There would be no action for the user to take, right? Then I don't see the use in warning them unnecessarily.